tools/scripts/prplMesh-platform-init-dummy.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+dbg() {
+    [ "$VERBOSE" == "true" ] && echo "$@"
+}
+
+err() {
+	echo -e '\033[1;31m'"$@"'\033[0m'
+}
+
+run() {
+    dbg "$*"
+    "$@" || exit $?
+}
+
+start_function() {
+    echo "$0: start"
+    if [ `id -u` -ne 0 ]; then
+        err "$0: This command must be run as root"
+        exit 1
+    fi
+    ip link add wlan0 type dummy
+    ip link add wlan2 type dummy
+    ip link add eth1 type dummy
```

Comment:
wlan0 and eth1 are interface names that might exist on the host. It would be better to replace these with names that are more likely to be unique.

Obviously, that means changing the agent config as well.

Suggested name: `pmt-ethN` for wired and `pmt-wlanN` for wireless. `pmt` stands for prplMeshTest. Note that the names can't be very long, there's a limit of 11 characters in some places.

Reply:
ok